usaspending_api/idvs/v2/views/activity.py

```diff
@@ -59,10 +59,18 @@
             ca.type not like 'IDV%'
             {hide_edges_awarded_amount}
         left outer join transaction_fpds tf on tf.transaction_id = ca.latest_transaction_id
-        left outer join recipient_lookup rl on rl.duns = tf.awardee_or_recipient_uniqu
-        left outer join recipient_profile rp on
+        left outer join recipient_lookup rl on (
+            (rl.uei is not null and rl.uei = tf.awardee_or_recipient_uei)
+            OR (rl.duns is not null and rl.duns = tf.awardee_or_recipient_uniqu)
+
+        )
+        left outer join recipient_profile rp on (
             rp.recipient_hash = rl.recipient_hash and
-            rp.recipient_level = case when tf.ultimate_parent_unique_ide is null then 'R' else 'C' end
+            rp.recipient_level = case
+                when coalesce(tf.ultimate_parent_uei, tf.ultimate_parent_unique_ide) is null
```

Comment:
I believe we heard from PO that the distinction of parentage is only through the UEI now. Not if a parent UEI _OR_ parent DUNS is provided.

Reply:
Removed reference to parent DUNS